frontend/src/static/js/main.js

```diff
@@ -1,3 +1,20 @@
+const store = new Vuex.Store({
```

Comment:
Better to put in a separate file. It is kind of standard.

Reply:
I intended to put it in a separate file but for our current structure, I failed to use import. So if in a separate file I have to use sth like window.store. I am not very sure if I am correct on this. Can tell me how to separate the files correctly? Thx in advance!

Reply:
Why can't you import the store? Did you export it?

Reply:
![image](https://user-images.githubusercontent.com/35621726/77844299-3425f980-71d8-11ea-86ff-6d0043d78776.png)
That's how I use vuex in another program.

Reply:
My js file is exactly the same. After that in the `main.js` need to have sth like `import store from *.js` right? Then I will have this error `main.js:1 Uncaught SyntaxError: Cannot use import statement outside a module`. I dont know what's wrong with the structure.

Reply:
> My js file is exactly the same. After that in the `main.js` need to have sth like `import store from *.js` right? Then I will have this error `main.js:1 Uncaught SyntaxError: Cannot use import statement outside a module`. I dont know what's wrong with the structure.

I played around with it. This should work if you change `script(src="static/js/main.js")` to `script(type="module" src="static/js/main.js")`.
Just use `import store from './store.js';` in `main.js`.